portal/static/portal/sass/styles.scss

```diff
@@ -1394,7 +1551,13 @@ td {
   .banner .col-sm-6 .banner--picture,
   .sub-nav--teacher,
   .sub-nav--student,
-  .form__choice {
+  .form__choice,
+  .onboarding__step--done:before,
+  .onboarding__step--done:after,
+  .onboarding__step--in-progress:before,
+  .onboarding__step--in-progress:after,
+  .onboarding__step--todo:before,
+  .onboarding__step--todo:after {
     display: none;
```

Comment:
This is clumsy.  With befores and afters, the only place they exist is the stylesheet, so you shouldn't need to hide them per breakpoint.  Instead, only define them for the breakpoints where you need them.